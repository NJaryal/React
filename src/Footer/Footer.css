.footerBg {
    background: #9ca4ab;
    width: 100%;
    display: block;
    text-align: center;
    color: #F65261;
    padding: .5rem;
}