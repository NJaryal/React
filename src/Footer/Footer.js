import React from 'react';
import styles from './Footer.css';

const Footer = () => (
  <div className={styles.footerBg}>
    <strong>netflix</strong>
      roulette
  </div>
);
export default Footer;
