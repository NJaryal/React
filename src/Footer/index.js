export { default } from 'src/Footer/Footer';

