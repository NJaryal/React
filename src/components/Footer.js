import React, { Component } from 'react'
class Footer extends Component {
    render() {
        return (
            <div className="footer">
            <strong>netflix</strong>roulette
            </div>
        );
    }
}
export default Footer;