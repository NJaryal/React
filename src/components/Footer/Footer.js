import React from 'react';
import './footer.css';

const Footer = () => (
  <div className="footer">
    <strong>netflix</strong>
      roulette
  </div>
);
export default Footer;
