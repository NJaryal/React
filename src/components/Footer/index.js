import React from 'react';
import './Footer.css';

const Footer = () => (
  <div className="footerBg">
    <strong>netflix</strong>
      roulette
  </div>
);
export default Footer;
