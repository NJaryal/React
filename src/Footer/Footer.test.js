import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

const setup = () => {
  const component = shallow(<Footer />);
  return component;
};

describe('Footer component', () => {
  let component;
  beforeEach(() => {
    component = setup();
  });

  it('should render Component', () => {
    expect(component).toMatchSnapshot();
  });
});
