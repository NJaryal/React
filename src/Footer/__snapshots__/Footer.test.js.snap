// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render Component 1`] = `
<div
  className="footerBg"
>
  <strong>
    netflix
  </strong>
  roulette
</div>
`;

exports[`Footer component should render without errors 1`] = `
<div
  className="footerBg"
>
  <strong>
    netflix
  </strong>
  roulette
</div>
`;
